GOVERNANCE.rst

```diff
@@ -0,0 +1,143 @@
+.. _governance:
+
+AMReX Governance
+================
+
+AMReX is led in an open governance model, described in this file.
+
+
+Steering Committee
+------------------
+
+Current Roster
+^^^^^^^^^^^^^^
+
+- Ann Almgren
+- John Bell (chair)
+- Andrew Myers
+- Weiqun Zhang
+
+Role
+^^^^
+
+Members of the steering committee (SC) can change organizational settings, do administrative operations such as rename/move/archive repositories, change branch protection rules, etc.
+
+SC members can call votes for decisions (technical or governance).
+
+The SC can veto decisions of the technical committee (TC) by voting in the SC.
+The TC can override a veto with a 2/3rd majority vote in the TC.
+Decisions are documented in developer meeting notes and/or on the GitHub repository.
+
+The SC can change the governance structure, but only in a unanimous vote.
+
+Decision Process
+^^^^^^^^^^^^^^^^
+
+Decisions of the SC usually happen in the developer meetings, via e-mail or public chat.
+
+Decisions are made in a non-confidential manner, by the majority of votes cast by SC members.
+
+Votes can be cast asynchronously, e.g., over a time period of 1-2 weeks.
+In tie situations, the chair of the SC acts as the tie breaker.
+
+Appointment Process
+^^^^^^^^^^^^^^^^^^^
+
+New members of the SC can be appointed by unanimous vote of the current SC members.
+
+SC members are expected to attend and contribute to regular developer meetings.
+
+SC members can resign or be removed by majority vote, e.g., due to inactivity, inappropriate or otherwise negative behavior, or other reasons.
+
+
+Technical Committee
+-------------------
+
+Current Roster
+^^^^^^^^^^^^^^
+
+- Ann Almgren
+- Marc Day
+- Candace Gilet
+- Kevin Gott
+- Axel Huebl
+- Andrew Myers
+- Weiqun Zhang
+- Michael Zingale
+
+See: `GitHub team <https://github.com/orgs/AMReX-Codes/teams/amrex>`__
+
+Role
+^^^^
+
+The technical committee (TC) is the core governance body, where, under normal operations, most ideas are discussed and decisions are made.
+Individual TC members can approve and merge code changes.
+TC members are expected to seek approval of another maintainer for their own changes, except under exigent circumstances.
+TC members lead - and weigh in on - technical discussions and, if needed, can call for a vote of the TC for a technical decision.
+TC members can merge/close PRs and issues, and moderate (including block/mute) bad actors.
```

Comment:
Use gerund forms? including blocking/muting?

Reply:
yes -- you are correct -- "(including blocking or muting)"